Sources/XcodeGenKit/PathKitExtension.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+import PathKit
+
+extension Path {
+
+    /// Treat this as a resource instead of a normal directory.
+    var isNonFolderDirectory: Bool {
```

Comment:
`isNonFolderDirectory` sounds a bit ambiguous. What about `isGroupDirectory`? That's how Xcode names directories in an Xcode project.

Reply:
Then you could just say `!myPath.isGroupDirectory`

Reply:
I understand, but I think it's depends on how you look at the problem.

From Xcode perspective, `isGroupDirectory` is maybe straight forward, but this specific functionality is seen outside of Xcode too. That's why I've put this as `PathKitExtension`. 

Also, I actually tried to rename it, and ended up with rather complicated code. (Needed additional `is a directory, but` condition to retrieve **non "public.folder" directory**.)

```swift
        let directories = children
            .filter { $0.isDirectory && $0.isGroupDirectory && $0.extension != "lproj" }
            .sorted { $0.lastComponent < $1.lastComponent }

        let filePaths = children
            .filter { $0.isFile || ($0.isDirectory && !$0.isGroupDirectory) }
            .sorted { $0.lastComponent < $1.lastComponent }
```

Therefore, I think it's better to define this way (`isNonFolderDirectory`) in whitelist manner.

Reply:
Done in 3aac581